extensions/spring-data-jpa/deployment/src/main/java/io/quarkus/spring/data/deployment/generate/SpringDataRepositoryCreator.java

```diff
@@ -163,7 +163,7 @@ private void createCustomImplFields(ClassCreator repositoryImpl, List<DotName> c
         for (String customImplClassName : customImplClassNames) {
             FieldCreator customClassField = repositoryImpl
                     .getFieldCreator("customImplClass" + (i + 1), customImplClassName)
-                    .setModifiers(Modifier.PRIVATE);
+                    .setModifiers(Modifier.PUBLIC); // done to prevent warning during the build
```

Comment:
Nice catch!

I wonder, if you remove the modifier completely, does it make the field package private and still remove the warning?

Reply:
> Nice catch!

Thanks.
 
> I wonder, if you remove the modifier completely, does it make the field package private and still remove the warning?

By default the modifier is `private`, so I think only removing the modifier not make the field package private.

Reply:
I don't really like the public field, but of course if there aren't any other alternatives, it's much better that having a build warning

Reply:
Likewise here. let's use `protected` :-)